src/components/LoadingShape/readme.md

```diff
@@ -0,0 +1,180 @@
+            <div className="rainbow-m-vertical_medium">
+                <LoadingShape shape="circle" style={shapeStyle} />
+            </div>
+            <strong className="rainbow-m-vertical_xx-small">solid</strong>
+        </div>
+
+        <div className="rainbow-flex rainbow-flex_column rainbow-justify_end rainbow-align_center">
+            <div className="rainbow-m-vertical_medium">
+                <LoadingShape shape="circle" variant="avatar" style={shapeStyle} />
+            </div>
+            <strong className="rainbow-m-vertical_xx-small">avatar</strong>
+        </div>
+
+        <div className="rainbow-flex rainbow-flex_column rainbow-justify_end rainbow-align_center">
+            <div className="rainbow-m-vertical_medium">
+                <LoadingShape shape="square" variant="image" style={shapeStyle} />
+            </div>
+            <strong className="rainbow-m-vertical_xx-small">image</strong>
+        </div>
+    </div>
+</div>
+```
+
+
+##### LoadingShape interactive example
+
+```js
+import React, { useState } from 'react';
+import { LoadingShape, RadioGroup, Input } from 'react-rainbow-components';
+
+const styles = {
+    maxWidth: 400,
+    margin: 'auto',
+};
+
+const shapeContainerStyles = {
+    height: '150px',
+    display: 'flex',
+    justifyContent: 'center',
+}
+
+const radioContainerStyles = {
+    display: 'flex',
+    justifyContent: 'space-evenly',
+};
+
+const inputStyle = {
+    width: '150px',
+}
+
+const shapeOptions = [
+    { value: 'rounded-rect', label: 'Rounded Rect' },
+    { value: 'square', label: 'Square' },
+    { value: 'circle', label: 'Circle' },
+];
+
+const variantOptions = [
+    { value: 'solid', label: 'Solid' },
+    { value: 'image', label: 'Image' },
+    { value: 'avatar', label: 'Avatar' },
+];
+
+const LoadingShapeExample = () => {
+    const [shape, setShape] = useState('rounded-rect');
+    const [variant, setVariant] = useState('solid');
+    const [width, setWidth] = useState(300);
+    const [height, setHeight] = useState(15)
+
+    const shapeStyle = {
+        height: `${height}px`,
+        width: `${width}px`,
+    };
+
+    const handleShapeChange = (event) => {
+        setShape(event.target.value);
+        if (event.target.value === 'circle' || event.target.value === 'square') {
+            setWidth(75);
+            setHeight(75);
+        } else {
+            setWidth(200);
+            setHeight(15);
+        }
+    };
+
+    return (
+        <div className="rainbow-p-vertical_x-large rainbow-p-horizontal_small" style={styles}>
+            <div className="rainbow-align-content_center rainbow-flex_wrap">
+                <Input
+                    label="Width"
+                    className="rainbow-p-around_small"
+                    style={inputStyle}
+                    value={width}
+                    onChange={event => setWidth(event.target.value)}
```

Comment:
width and height inputs should use ```type="number"``` in order to change the value with the up and down arrows

Reply:
I missed that, thanks.